xml/security_ssh.xml

```diff
@@ -550,30 +533,109 @@ cd path                            Change remote directory to 'path'
   </note>
 
   <sect2 xml:id="sec-ssh-authentic-gen-key">
-   <title>Generating an SSH key</title>
-   <procedure>
-    <step>
+   <title>Generating SSH keys</title>
+   <para>
+     There are several key types to choose from: DSA, RSA, ECDSA, ECDSA-SK, 
+     Ed25519, and Ed25519-SK. DSA was deprecated in OpenSSH 7.0 and should 
+     not be used. RSA is the most universal and is accepted by older clients. 
```

Comment:
... accepted by older servers.

Reply:
This sentence relates to the Note, "Older clients may be unsafe". Presumably the OpenSSH server admin has control of their own server, and may have to deal with older SSH clients they do not control. Does that make sense?

Reply:
Where are my manners, thank you for the review!

Reply:
The idea is correct, just the wording is confusing. "and is used by older clients" would be an alternative